pkg/internal/transform/route/harvest/harvester.go

```diff
@@ -58,10 +56,10 @@ func (e *HarvestError) Error() string {
 func NewRouteHarvester(cfg *services.RouteHarvestingConfig, disabled []string, timeout time.Duration) *RouteHarvester {
```

Comment:
@grcevski so the config is still a string[] but we compare it to the string enum?
maybe we can create `MarshalText` and `UnmarshalText` on `InstrumentableType` to allow users to specify it and have config type safety?